app/mixins/editor-base-controller.js

```diff
@@ -41,7 +41,7 @@ export default Mixin.create({
     notifications: injectService(),
     clock: injectService(),
     slugGenerator: injectService(),
-
+    wordCount: 0,
```

Comment:
The `gh-editor` component is the wrapper for the editor pane, it contains the header, title, and editor and seems to be the perfect place for this. The editor base controller will be going away with much of the functionality in it moved to `gh-editor` or into more appropriate places .